cmd/lifecycle/analyzer.go

```diff
@@ -224,3 +223,21 @@ func (a *analyzeCmd) platformAPIVersionGreaterThan06() bool {
 func (a *analyzeCmd) restoresLayerMetadata() bool {
 	return !a.platformAPIVersionGreaterThan06()
 }
+
+func (aa *analyzeArgs) ReadableRegistryImages() []string {
+	var readableImages []string
+	if !aa.useDaemon {
+		readableImages = append(readableImages, aa.previousImageRef, aa.runImageRef)
+	}
+	return readableImages
+}
+func (aa *analyzeArgs) WriteableRegistryImages() []string {
+	var writeableImages []string
+	if aa.cacheImageRef != "" {
+		writeableImages = append(writeableImages, aa.cacheImageRef)
+	}
+	if !aa.useDaemon {
+		writeableImages = append(writeableImages, append([]string{aa.outputImageRef}, aa.additionalTags...)...)
```

Comment:
Should we add the `outputImageRef` to the PR description?

Reply:
Done